### PR TITLE
docs(deploy): add tencent cloud webify

### PR DESCRIPTION
**Pull request by starkwang**
## Check List

**Please read and check followings before submitting a PR.**

- [ ] I want to publish my theme on Hexo official website.
  - [ ] I have read the [theme publishing doc](https://hexo.io/docs/themes#Publishing).
  - [ ] `name` is unique.
  - [ ] `link` URL is correct.
  - [ ] `preview` URL is correct.
  - [ ] `preview` URL web site is rendered correctly.
  - [ ] Add a screenshot to `source/themes/screenshots`.
  - [ ] Screenshot filename is same as value of `name`.
  - [ ] Screenshot size is `800 * 500`.
  - [ ] Screenshot file format is `png`.
- [ ] I want to publish my plugin on Hexo official website.
  - [ ] I have read the [plugin publishing doc](https://hexo.io/docs/plugins#Publishing).
  - [ ] `name` is unique.
  - [ ] `link` URL is correct.
- [x] Others (Update, fix, translation, etc...)
  - Languages:
  - [ ] `en` English
  - [ ] `ko` Korean
  - [ ] `pt-br` Brazilian Portuguese
  - [ ] `ru` Russian
  - [ ] `th` Thai
  - [x] `zh-cn` simplified Chinese
  - [ ] `zh-tw` traditional Chinese

### CODE REVIEW

**Review by tomap — APPROVED** [review]

File: source/zh-cn/docs/one-command-deployment.md [hunk 1]
```diff
@@ -300,6 +300,20 @@ $ hexo generate —deploy && bip deploy
...
+## 腾讯云 Webify
+
+[腾讯云 Webify](https://webify.cloudbase.net/) 支持从 Git 仓库直接部署您的 Hexo 应用。
+
+进入 [Webify 新建应用页面](https://console.cloud.tencent.com/webify/new)，根据选择您代码仓库所在的 Git 平台（Github、Gitlab 或者 Gitee 码云），完成授权流程后，便可导入仓库。
+
+填入应用配置如下：
+
+- 构建命令填入 `npm run build`
+- 输出目录填入 `dist`，
```
Comment (stevenjoezhang) [inline source/zh-cn/docs/one-command-deployment.md:312]: 输出是 dist 还是 public？

File: source/zh-cn/docs/one-command-deployment.md [hunk 2]
```diff
@@ -300,6 +300,20 @@ $ hexo generate —deploy && bip deploy
...
+- 安装命令填入 `npm install`
+
+应用创建之后，等待构建、部署完毕，便可以通过应用的默认域名（`.app.tcloudbase.com`）来访问应用。如 https://my-hexo-site-0g2fpeyz0f499162-1255679239.ap-shanghai.app.tcloudbase.com/
```
Comment (stevenjoezhang) [inline source/zh-cn/docs/one-command-deployment.md:315]: 示例链接具有时效性，建议移除

**Comment by uiolee** [comment]
Since #133, it is more appropriate to add the English document first. 

Since it's been idle for so long and not updated, I'd like to close this pr.

File: source/zh-cn/docs/one-command-deployment.md (hunk 1 above)
Comment (uiolee) [reply source/zh-cn/docs/one-command-deployment.md:312]: It defaults to `public`

**Pull request closed without merging**